mopidy/audio/actor.py

```diff
@@ -481,6 +482,7 @@ def _setup_audio_sink(self):
         audio_sink.add_pad(ghost_pad)
 
         self._playbin.set_property('audio-sink', audio_sink)
+        self._queue = queue
```

Comment:
Why not set `self._queue = Gst.ElementFactory.make('queue')` on line 467? There's no returns or early exits in this method, so it'll have the same effect.